### DISCUSSION TITLE
Stuck visual mode selection

### DISCUSSION THREAD

**Discussion by guivictorr** [LazyVim/LazyVim#4435] [Q&A] [answered]
can someone help me find out the reason behind this ? I'm in normal mode but the selection from visual mode still remains

<img width="409" alt="Screenshot 2024-09-13 at 14 13 05" src="https://github.com/user-attachments/assets/d11d6bb8-d983-4305-841b-4c1f99dc8f45">

**Comment by dpetka2001** [comment, marked as answer]
This is an issue with Neovim builtin `vim.snippet` engine. Not a LazyVim issue. As a workaround just go into `insert` mode inside the snippet region and press `<Tab>` until it exits the snippet region.

**Reply by guivictorr** [reply]
Thanks for help me with this
